ci_framework/roles/build_containers/tasks/install.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Install required packages
+  tags:
+    - packages
+  become: true
+  ansible.builtin.package:
+    name:
+      - python3-devel
+      - python3-pip
+      - podman
+      - buildah
+    state: latest
+
+- name: Get tcib from repository
+  ansible.builtin.git:
+    accept_hostkey: true
+    dest: "{{ cifmw_build_containers_basedir }}/tmp/tcib"
+    repo: "{{ cifmw_build_containers_tcib_src }}"
+
+  # FIXME(Chandan/arxcruz): Skip usage of sudo installation
```

Comment:
the tcib docs is updated now. https://github.com/openstack-k8s-operators/tcib#building-images-with-local-changes it will work without sudo.